template.env

```diff
@@ -3,7 +3,7 @@ DEFAULT_GROUP=Home
 ENV=False
 API_PORT=9000
 API_DOCS=True
-DB_TYPE=sqlite
+DB_URL=sqlite:///data/mealie.db
```

Comment:
This should default to None and allow Pydantic to properly set the URL accordingly.